mars/utils.py

```diff
@@ -453,8 +453,8 @@ def get_expr_module(op):
     try:
         return _expr_modules[module_name]
     except KeyError:
-        # tensor.expressions and dataframe.expressions have method concat_tileable_chunks
-        expr_module_name = '.{0}.expressions'.format(module_name)
+        # tensor.utils and dataframe.utils have method concat_tileable_chunks
+        expr_module_name = '.{0}.utils'.format(module_name)
```

Comment:
A little strange for me specifying ``utils`` here. Maybe importing ``concat_tileable_chunks`` and other utility functions into ``mars.tensor`` and ``mars.dataframe`` and use these root dirs directly is better.